kolibri_instant_schools_plugin/assets/src/views/about/views/about/faq.html

```diff
@@ -21,7 +21,7 @@ <h2>What is Instant Schools?</h2>
 
     <h2>What education channels are available?</h2>
     <p>
-      Available education channels for Learners and Teachers are displayed in Recommended and Topics tabs on the main Learn page. Current channels include:
+      Available education channels for Learners and Teachers are displayed in Recommended and Channels tabs on the main Learn page. Current channels include:
     </p>
 
     <h3>Learner channels</h3>
```

Comment:
Sikana and Touchable Earth should be removed from list under Learner channels. TESSA should be removed from list under Teacher channels

Reply:
addressed comments - ready for re-review